tools/integration_tests/util/setup/setup.go

```diff
@@ -234,3 +234,18 @@ func LogAndExit(s string) {
 	log.Print(s)
 	os.Exit(1)
 }
+
+// Clean the mounted directory
+func CleanMntDir() {
```

Comment:
what's the difference between CleanMntDir and os.removeall ? why are we making this shift now?

Reply:
So before we were not catching the error for os.removeall. os.RemoveAll(mntDir) will remove content of directory and directory itself, but as we can't remove mntDir as we mounted it(it was cleaning the mntDir but at the end throwing error that we can't remove mntDir)

In CleanMntDir() --> I am just cleaning mntDir content and not mntDir itself.

Reply:
Swetha suggested that we should not ignore any error or file handler. That's why I am capturing it every place.